Added new focus icons / Added new POIs in Bastion

Dathwada opened the pull request:
Added two new icons for focused Treasures and AnimaShards (green glow chest and shard)

Krovikan commented:
I am an idiot.

The better way to introduces the changes to enUS base in esES is copy the file again (instead of my additons and deletions). For do this, is better wait that you create a esES file someday.

Sorry for pull request.

Dathwada committed:
Added new focus icons / Added new POIs in Bastion

Added two new icons for focused Treasures and AnimaShards (green glow chest and shard)

zarillion commented on core/nodes.lua:
```diff
@@ -322,11 +322,17 @@ end
     group = 'treasures'
 })
 
+function Treasure.getters:icon ()
+    if self._focus then
+        return 'chest_gray_green_glow'
+    end
+        return 'chest_gray'
+end
```
This getter doesn't work exactly as you expect. The way `class.lua` is implemented, if the attribute is not present on the top-level object, it checks for getters all the way down the class ancestry and runs the first one it finds. If it doesn't find one, it then checks all the way down the class ancestry for a static class attribute value.

This change breaks the `Supply()` subclass of `Treasure()`, which looks like this:

```lua
local Supply = Class('Supply', Treasure, {
    icon = 'star_chest',
    scale = 2
})
```

The value `star_chest` will never be used, because the parent class has a getter function set for that attribute. On this fork all Nazjatar supply chests and large locked caches in Vale/Uldum display as the plain treasure icon instead of the `star_chest`.

For now, the simplest fix is to also define a getter function for the `Supply` cache that overrides the one defined for `Treasure` so the icon is restored.

```lua
function Supply.getters:icon ()
    return 'star_chest'
end
```

Long-term I would like to fix `class.lua` to search for a static attribute,  then a getter at each class inheritance level instead of the current behavior. The structure of it is complex enough that without a rigorous set if tests its hard to get all the metaclassing tables exactly right.

zarillion reviewed (commented):
The rest of this looks good. Fix the `Supply()` class and it should be ready to merge.

Dathwada committed:
Update nodes.lua

Fix for Supply()
